server/tests/test_responses_api.py

```
@@ -0,0 +1,103 @@
+    print(f"Testing Responses API with model: {model_spec}")
+    
+    # Pre-load the model to avoid timeout in test
+    try:
+        from server.cache_utils import get_model_path
+        model_path, _, _ = get_model_path(model_spec)
+        if not model_path or not model_path.exists():
+             print(f"Skipping test, model {model_spec} not found in cache.")
+             return
+             
+        get_or_load_model(model_spec)
+    except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)